pipelines/hello_world/README.md

```diff
@@ -10,7 +10,7 @@ A companion tutorial will cover how to run pipelines on Kubeflow Pipelines, enab
 
 ### Prerequisites
 
-- JupyterLab 2.x with the Elyra extension v1.1 (or newer) installed
+- JupyterLab 2.x with the Elyra extension v1.1 (or newer) installed. Refer to the Elyra repository for [deployment options](https://github.com/elyra-ai/elyra#try-elyra).
```

Comment:
Should we actually link them to : https://elyra.readthedocs.io/en/latest/getting_started/installation.html

Reply:
We could also do that. I linked to the readme because it also had deployment instructions for docker, which are not in the installation guide. Not sure if that's intentional or not. If you prefer to associate the tutorial more closely with a native installation then I'll change the link as requested.

Reply:
I made the requested change.  We can always add the docker content etc to that document, like it was done for JupyterLab.